packages/@okta/vuepress-site/docs/guides/custom-widget/main/index.md

```diff
@@ -50,16 +50,14 @@ The **Custom Sign-In Page** offers basic and advanced customization options to c
 
 To access this page:
 
-> **Note:** If you've enabled [Early Access (EA) multibrand customization](https://help.okta.com/okta_help.htm?type=oie&id=csh-branding), your Admin Console navigation is different. See parenthetical notes.
-
-In the Admin Console, go to **Customizations** > **Branding** then click **Edit** in the **Sign-in page** section. (EA users: go to **Customizations** > **Brands**, and then select the brand you want.)
+In the Admin Console, go to **Customizations** > **Branding** then click **Edit** in the **Sign-in page** section. (If you enabled multibrand customization, go to **Customizations** > **Brands**, and then select the brand you want.)
```

Comment:
general comment about. I see this is like this in sections below as well.
> (If you enabled multibrand customization, go to **Customizations** > **Brands**, and then select the brand you want.).

there is additional step here after selecting the brand. how about this? or something similar

> (If you enabled multibrand customization, go to **Customizations** > **Brands**, and then select the brand you want. in the **Pages** tab, click **Configure** for the  **Sign-in Page**)

Reply:
Fixed. Thanks!